pkg/rolie/clone.go

```diff
@@ -108,11 +110,17 @@ func (f *fetcher) getRemoteResourceRaw(URI string) (io.ReadCloser, error) {
 }
 
 func (f *fetcher) storeLocally(URI string, content []byte) error {
-	filepath, err := f.filepath(URI)
+	path, err := f.filepath(URI)
 	if err != nil {
 		return err
 	}
-	return ioutil.WriteFile(filepath, content, 0777)
+
+	dirPath := filepath.Dir(path)
+	err = os.MkdirAll(dirPath, os.FileMode(0755))
+	if err != nil {
+		return err
+	}
+	return ioutil.WriteFile(path, content, 0777)
```

Comment:
@isimluk I think that I missed this one too... should this be 0644?

Reply:
Fixed! thx